src/js/components/sharedComponents/CheckboxTreeLabel.jsx

```diff
@@ -0,0 +1,33 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    value: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.array
+    ]),
+    label: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.array
+    ])
+};
+
+const CheckboxTreeLabel = ({
+    value,
```

Comment:
Can this HTML be more semantic? IE <label>?

Also thinking that we should try and make this as re-usable as possible.

Reply:
Porque?

Reply:
no because I cannot access the input associated with that label to pass it an id